.github/workflows/tests.yml

```
@@ -1,35 +1,39 @@
----
-
 name: Tests
 
 on:
   push:
-    branches:
-      - master
   pull_request:
-    branches:
-      - master
+  workflow_dispatch:
 
 jobs:
   build_doc:
-    name: 'Build cpython doc'
+    name: Build CPython documentation
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-python@v2
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
         with:
-          python-version: 3.8
-      - name: clone docsbuild scripts
+          python-version: 3
+      - name: Clone docsbuild scripts
         run: |
           git clone https://github.com/python/docsbuild-scripts/
-      - name: setup requirements
+      - name: Setup requirements
```

Nit: "setup" is a noun, "set up" for the verb form:
```suggestion
      - name: Set up requirements
```